Check/Codebase/PhpCodeSniffer.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * @file
+ * Contains \SiteAudit\Check\Codebase\PhpCodeSniffer.
+ */
+
+use Symfony\Component\Process\Process;
+/**
+ * Class SiteAuditCheckCodebasePhpCodeSniffer.
+ */
+class SiteAuditCheckCodebasePhpCodeSniffer extends SiteAuditCheckAbstract {
+  /**
+   * Implements \SiteAudit\Check\Abstract\getLabel().
+   */
+  public function getLabel() {
+    return dt('PHP Code Sniffer Violations');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getDescription().
+   */
+  public function getDescription() {
+    return dt('Run phpcs on custom code.');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultFail().
+   */
+  public function getResultFail() {}
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultInfo().
+   */
+  public function getResultInfo() {
+    if (isset($this->registry['phpcs_path'])) {
+      return dt('Cannot find phpcs in path. Make sure that phpcs is present in the PATH or run composer install inside the site_audit directory to install all the dependencies');
```

Comment:
Inaccurate, `cannot find phpcs in site_audit installation.` then provide instructions in the `getAction` method.